sdk/src/types/block/core/basic.rs

```diff
@@ -84,10 +84,42 @@ impl BasicBlockBodyBuilder {
         })
     }
 
+    /// Finishes the builder into a [`BasicBlockBody`] with the minimum amount of mana required for the block to get
+    /// accepted by the network.
+    pub fn finish_with_minimum_mana_amount(
```

Comment:
Should this be part of the block builder?

Reply:
What do you mean exactly?

Reply:
Yes

Reply:
So then this should probably be part of that process rather than a fn in the client

Reply:
oh man I was clearly confused

Reply:
shouldn't this all just be `finish` though? I mean it's invalid to not set this right

Reply:
I would very much prefer `finish` yeah. It can't not be set yes, but it can be set to a specific value or to minimum, like the storage deposit.

Reply:
Maybe a bit more annoying than storage deposit as it needs 2 params though

Reply:
okay ... changing it

Reply:
Done